docs/guides/communication.md

```diff
@@ -14,16 +14,16 @@ You can invoke:
 
 An invocation is defined in a state node's configuration with the `invoke` property, whose value is an object that contains:
 
-- `src` - the source of the machine to invoke, which can be:
+- `src` - the source of the service to invoke, which can be:
   - a machine
   - a string, which refers to a machine defined in this machine's `options.services`
   - a function that returns a `Promise`
   - a function that returns a callback handler, which sends events to the parent via the callback (example below)
 - `id` - the unique identifier for the invoked service
-- `forward` - (optional) `true` if all events sent to this machine should also be sent (or _forwarded_) to the invoked child machine (`false` by default)
-- `data` - (optional) an object that maps properties of the child machine's [context](./context.md) to a function that returns the corresponding value from the parent machine's `context`.
 - `onDone` - (optional) the [transition](./transitions.md) to be taken when the child machine reaches its [final state](./final.md)
-- `onError` - (optional) the transition to be taken when the child machine encounters an execution error.
+- `onError` - (optional) the transition to be taken when the child machine encounters an execution error
+- `forward` - (optional, used only for machines) `true` if all events sent to this machine should also be sent (or _forwarded_) to the invoked child machine (`false` by default)
```

Comment:
```suggestion
- `forward` - (optional) `true` if all events sent to this machine should also be sent (or _forwarded_) to the invoked child (`false` by default)
```

This will also be sent to invoked callbacks (and duplex Observables in a future relase)